lib/Target/SubstraitPB/Import.cpp

```diff
@@ -140,6 +141,32 @@ static mlir::FailureOr<CrossOp> importCrossRel(ImplicitLocOpBuilder builder,
   return builder.create<CrossOp>(leftVal, rightVal);
 }
 
+static mlir::FailureOr<UnionDistinctOp>
+importSetRel(ImplicitLocOpBuilder builder, const Rel &message) {
+  const SetRel &setRel = message.set();
+
+  assert(setRel.inputs_size() == 2 && "expected two inputs");
```

Comment:
I think a failure is more appropriate, here and below. An `assert` is something that shouldn't happen, in particular, not due to user-provided input, and is mainly there for documentation for readers of the code. (Note that assertions are not tested if the project is compiled in release mode.) Here, the problem comes from the structure of a user-provided file and the user may want to know about that problem.

BTW, since we'll remove that limitation soon, you don't need to write a test case for that error message.